docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,205 @@
+---
+id: account-recovery-password-reset
+title: Account recovery and password reset
+---
+
+```mdx-code-block
+import RenderFlow from "@site/src/theme/Code/RenderFlow"
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+```
```

Comment:
Doesn't hurt either, does it? :)

And it helps with readability because it's highlighted as one block in my editor and thus separated from the actual text. So maybe we should start doing that in the other docs as well:

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/5540255/199472381-aa038319-bb42-4bc5-9bba-e777a974855a.png">

Reply:
Sure, no harm in that. There are lots of other documents that don't do that already in our docs set (even in this PR) that don't do that. I'm trying to introduce consistency to curb the chaos that sometimes governs our docs. For that reason, I'd like to see that wrapping removed, pls :)

Reply:
> I don't think the import themselves must be wrapped - haven't seen this in any other docs.

They do need to be wrapped @tomekpapiernik . Please check the README of this repository ;) All code snippets are wrapped there. It's quite important to avoid problems in rendering.

https://github.com/ory/docs#import-markdown

Reply:
Okay, so this was ignored in many cases. Gonna have to make an effort to fix that in the repo. Will update this PR.